Tests/LibCompress/TestBrotli.cpp

```diff
@@ -75,3 +75,8 @@ TEST_CASE(brotli_decompress_duckduckgo_css2)
 {
     run_test("s2049.css");
 }
+
+TEST_CASE(brotli_decompress_cloudflare_html)
+{
+    run_test("cloudflare.html");
```

Comment:
You've extracted this nicely, but is cloudflare's HTML something we can legally include? I don't think so. Let's either use a website that's explicitly BSD compatible, or one of our own static sites, like a birthday announcement post.

Reply:
Yeah it might be a bit tricky, there are also some CSS files from DuckDuckGo in the tests too.

The problem is that this specific compressed file caused a problem with the implementation, while I could not reproduce it with other files.

In the end might just remove these tests, or replace them with actual HTTP requests, so the files are not included in our repository.

Reply:
How about you minimize the cloudflare problem and create an artificial test that's far enough from their HTML it doesn't cause problems?

Regarding DuckDuckGo, that also looks like a problem even though they would probably care less.

Reply:
I have removed all of the test cases that might be a possible copyright problem.
Instead I'm using a few of the SerentiyOS website pages, the Katica font file, and a tar of the Userland source code.